data/hai.txt

```diff
@@ -103,6 +146,50 @@ fleet "Large Hai"
 		"Shield Beetle (Pulse)"
 		"Shield Beetle (Tracker)"
 		"Lightning Bug" 2
+	variant
+		"Shield Beetle (Tracker)
+		"Centipede" 3
+	variant 2
+		"Water Bug" 3
+		"Centipede" 2
+	variant
+		"Centipede" 3
+		"Centipede (Pulse)" 2
+	variant 3
+		"Centipede" 2
+		"Lightning Bug" 3
+		"Water Bug"
+	variant 2
+		"Aphid" 2
+		"Lightning Bug"
+		"Centipede"
+	variant 2
+		"Centipede"
+		"Shield Beetle (Pulse)" 2
+		"Shield Beetle (Tracker)"
+	variant
+		"Centipede" 2
+		"Lightning Bug" 2
+		"Aphid" 3
+	variant 1
+		"Geocoris"
+		"Shield Beetle (Pulse)" 2
+		"Shield Beetle (Tracker)"
+	variant 1
+		"Geocoris"
+		"Shield Beetle (Pulse)" 2
+		"Grasshopper"
+		"Grasshopper (Pulse)"
+	variant 1
+		"Geocoris"
+		"Shield Beetle (Pulse)" 
+		"Shield Beetle (Tracker)"
+		"Grasshopper (Pulse)"
+		"Grasshopper (Tracker)"
+	variant 1
+		"Geocoris" 2
+		"Shield Beetle" 1
```

Comment:
You don't need to include the 1 for the number of ships

Reply:
if the variant weight is 1, it doesnt need to be specified either